source/index.html.md

```diff
@@ -149,6 +149,39 @@ table | none | The ID of an attribute table to limit results to
 
 Dimensions store the chart of accounts for a company and outline how their data is structured. All companies have top level dimensions called Account, Scenario, Department, and Time. They also can have custom dimensions.
 
+## Create A Dimension
+
+```python
+import requests
+
+r = requests.get('<url>', headers={
+    'Authorization': 'example-access-token'
+})
+r.json()
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+    "name": "New Dimension Name",
+	"parent": 123
+}
+```
+
+This endpoint creates a dimension.
+
+### HTTP Request
+
+`POST https://api.cubesoftware.com/v1/dimensions`
```

Comment:
Might say a sentence or two about the default values when creating a dimension - type, write protection. Can parent be None, etc.